src/main/java/com/quantumsoft/qupathcloud/entities/Projects.java

```diff
@@ -18,15 +18,32 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * The Projects class contains a list of {@link com.quantumsoft.qupathcloud.entities.Project}
+ * retrieving from the server.
+ *
```

Comment:
Projects contains a list of {@link com.quantumsoft.qupathcloud.entities.Project}
 retrieved from the server.